data/map.txt

```diff
@@ -26965,6 +27082,14 @@ planet Echo
 	bribe 0
 	security 0.9
 
+planet Eldergrasp
+	attributes station hai retirement
+	landscape land/space6
+	description `A visitor to this station will immediately notice that the gravity is lower than most planets by about a third. There is a generously appointed reception area with an orderly on duty who informs you that, unless you have family at this respite facility, you will not be permitted to wander the station.`
+	description `The spaceport is virtually nonexistent, with just a single cafeteria, a fuel pump for the desperate, and a dedicated loading area for loading and unloading this retirement home's regular deliveries. Despite it's small size though, everything is in excellent condition.`
```

Comment:
Don't you need a spaceport node to refuel?
```suggestion
	description `The spaceport is virtually nonexistent, with just a single cafeteria, a fuel pump for the desperate, and a dedicated loading area for loading and unloading this retirement home's regular deliveries. Despite its small size though, everything is in excellent condition.`
```

Reply:
So I want a spaceport (and ideally a job board) but not trading on the retirement stations. Turns out I can't do that so when I moved the second line into description I was sort of going with the implication that you're not "the very desperate".